code/modules/awaymissions/capture_the_flag.dm

```diff
@@ -48,7 +48,8 @@
 		STOP_PROCESSING(SSobj, src)
 
 /obj/item/weapon/twohanded/ctf/attack_hand(mob/living/user)
-	if(!user)
```

Comment:
You need to check that it's actually a mob/living with if(istype(user)) just specifying the type path in the variable argument is never enough.

Does is_ctf_target do that?